client/components/app-promo/index.jsx

```diff
@@ -69,15 +75,26 @@ export default React.createClass( {
 			return null;
 		}
 
-		var promo_link = 'https://apps.wordpress.com/desktop/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
-		var element = (
+		const promo_link = 'https://apps.wordpress.com/desktop/?ref=promo_' + this.props.location + '_' + this.state.promo_item.promo_code;
+		const element = (
 			<div className="app-promo">
```

Comment:
Yeah I think there's a bug with the rule currently when assigning an element to a variable outside the `return` statement of a `render`. Which begs the question, why not just `return` here instead of creating an `element` variable?